awpy/plot/plot.py

```diff
@@ -22,14 +23,19 @@
 def plot(  # noqa: PLR0915
     map_name: str,
     points: Optional[List[Tuple[float, float, float]]] = None,
+    is_lower: Optional[bool] = False,
```

Comment:
What if this parameter were reworked into a "lower fraction"? As in, we assume that upper points are plotted with alpha = 1.0, and then lower are lower_frac*alpha ? This would allow for a more expressive form of what it appears you are going for.

Reply:
Which, we may also want to offer a param for the "dead player alpha" that we default to 0.15, too

Reply:
Good idea!

The only thing to note is that when `is_lower = True`, it also changes the image to the lower part of the map, but then again, you can control this via the `map_name` argument.

Reply:
Now that I look at this, perhaps we want to just have a generic `game_to_pixel(tuple)`. That tuple can be an (x,y) pair. What do you think?